src/Trie.mo

```diff
@@ -326,6 +352,10 @@ module {
       replace<K, V>(t, k, k_eq, ?v)
     };
 
+  /// get the value of the given key in the trie, or return null if nonexistent
+  public func get<K, V>(t : Trie<K, V>, k : Key<K>, k_eq : (K, K) -> Bool) : ?V =
+     find<K, V>(t, k, k_eq);
+
   ///  find the given key's value in the trie, or return null if nonexistent
   public func find<K, V>(t : Trie<K, V>, k : Key<K>, k_eq : (K, K) -> Bool) : ?V =
     label profile_trie_find : (?V) {
```

Comment:
It would be nice to get rid of these profiling blocks now....

Reply:
Done.